src/gatsby-browser.js

```diff
@@ -2,30 +2,27 @@ const injectPinterestScript = (tall, round) => {
   function addJS() {
     const s = document.createElement(`script`);
     s.type = `text/javascript`;
-    s.setAttribute("async", "async");
-    s.setAttribute("defer", "defer");
-      s.setAttribute("data-pin-hover", "true");
-    if(tall) {
-      s.setAttribute("data-pin-tall", "true");
+    s.setAttribute('async', 'async');
+    s.setAttribute('defer', 'defer');
+    s.setAttribute('data-pin-hover', 'true');
+    if (tall) {
+      s.setAttribute('data-pin-tall', 'true');
     }
-    if(round) {
-      s.setAttribute("data-pin-round", "true");
+    if (round) {
+      s.setAttribute('data-pin-round', 'true');
     }
-    s.setAttribute("src", "https://assets.pinterest.com/js/pinit.js");
+    s.setAttribute('src', 'https://assets.pinterest.com/js/pinit.js');
     document.getElementsByTagName(`head`)[0].appendChild(s);
   }
   addJS();
 };
 
-let injectedPinterestScript = false
+let injectedPinterestScript = false;
 
 exports.onRouteUpdate = (args, pluginOptions) => {
   if (document.querySelector('[data-pin-do]') !== null) {
     if (!injectedPinterestScript) {
-      const {
-        tall = true,
-        round = false,
-      } = pluginOptions;
+      const { tall = true, round = false } = pluginOptions;
```

Comment:
`kcd-scripts` [is using](https://github.com/kentcdodds/kcd-scripts/blob/45c2bbc2b898872245bb4abe044a55e867d12de8/src/config/eslintrc.js#L5) `eslint-config-kentcdodds` for it's ESLint config, [which is supporting ES6](https://github.com/kentcdodds/eslint-config-kentcdodds/blob/59d9515949df63657ac035cd31e47b86921c95ad/index.js#L11), so I think there's something wrong on your side.

Reply:
Just killed and restarted VSCode, still getting the warnings. Is it because I don't have eslint enabled globally? What about making an `.eslintrc` that just exports kcd's config, just like you did for Prettier?

Reply:
I needed to remove the `babel/quotes` rule, so this should be fine by now normally. 🙂